src/app/researcher/study/request/[memberSlug]/study-proposal-form.tsx

```diff
@@ -44,27 +44,35 @@ export const StudyProposalForm: FC<{
     const { user } = useUser()
 
     return (
-        <Paper p="md">
-            <Title order={4}>Study Proposal</Title>
-            <Divider my="sm" mt="sm" mb="md" />
+        <Paper pt="sm" pl="xl" pr="lg" pb="lg">
+            <Title order={4} pr="lg" pb="sm">
+                Study Proposal
+            </Title>
+            <Divider my="md" />
             <Text mb="md">
                 This section is here to help you submit your study proposal. Consider providing as much detail as
                 possible to ensure the Reviewer has all the information needed to make an informed decision.
             </Text>
 
             <Grid>
                 <GridCol span="content">
-                    <Stack gap="xl">
-                        <Text fw="bold">Study Title</Text>
-                        <Text mt="md" fw="bold">
+                    <Stack gap="xs">
```

Comment:
Just trying to catch this as early as possible, but read [my comment here](https://github.com/safeinsights/management-app/pull/150#issuecomment-2815334496) i think this is my fault and partly the nature of all of us being set loose on tickets that possibly overlap... We have 3 PRs that are now all modifying the layout of the form and they're all different which will cause mega conflicts

this isn't your fault, i think i can take most of the responsibility for not having caught this in tickets overlapping work but we should try to figure out how to handle this 😭

Reply:
maybe all 3 of us can circle up on monday when everyone's here -- we can figure it out pretty quick i bet 🤞